.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- Keep your project professional add into your gitignore external files like: .DS_Store, .idea or node_modules.
(In your root directory, you have an DS_Store file)

Reply:
@carlos-ssh Let me make those changes right away. Thank you.